### ISSUE TITLE
0.7.1 tar file wrong symlink for electron build

### ISSUE THREAD

**Issue by j0ack** [glowing-bear/glowing-bear#1068]
It seems the 0.7.1 tarfile does contains a wrong symlink for electron build

```
 /tmp/glowing-bear-0.7.1/glowing-bear/min.map -> ../min.map
```

Where a `/tmp/glowing-bear-0.7.1/glowing-bear/min.js.map` is present.

**Comment by lorenzhs** [comment]
Thanks, the next release should contain the fixed symlink!

**Comment by lorenzhs** [comment]
Fixed in 0.7.2, which I released just now
